lib/modules/platform/types.ts

```diff
@@ -288,7 +288,20 @@ export interface Platform {
     branchName: string,
     internalChecksAsSuccess: boolean,
   ): Promise<BranchStatus>;
-  getBranchPr(branchName: string, targetBranch?: string): Promise<Pr | null>;
+  getBranchPr(
+    branchName: string,
+    /**
```

Comment:
move to tsdoc style comment above the function, so it's visible at call site in editor like vscode